Dockerfile

```diff
@@ -1,23 +1,27 @@
 FROM ubuntu:14.04
 
+ENV SHELL="/bin/bash"
```

Comment:
Just curious, which part of this required a bash shell?

Reply:
The Drakefile uses bash syntax, so drake fails